main.go

```diff
@@ -167,19 +171,17 @@ func main() {
 				webhookFactory.SetExternalUpdate(createArgusSynchronizer(argus, logger))
 				lc.Append(fx.Hook{
 					OnStart: func(context.Context) error {
-						// wait for DNS to propagate before subscribing to SNS
-						if err = webhookFactory.DnsReady(); err == nil {
-							//TODO: If we know with certainty this is a docker-compose specific hack, we could add a delay elsewhere or
-							//only run this case when running from our test cluster?
-							time.Sleep(time.Second * 5)
-							logging.Info(logger).Log(logging.MessageKey(), "server is ready to take on subscription confirmations")
-							webhookFactory.PrepareAndStart()
-							return nil
-						} else {
-							logging.Error(logger).Log(logging.MessageKey(), "Server was not ready within a time constraint. SNS confirmation could not happen",
+						if err := webhookFactory.DnsReady(); err != nil {
+						       logging.Error(logger).Log(logging.MessageKey(), "Server was not ready within a time constraint. SNS confirmation could not happen",
 								logging.ErrorKey(), err)
 							return err
 						}
+						//TODO: If we know with certainty this is a docker-compose specific hack, we could add a delay elsewhere or
+						//only run this case when running from our test cluster?
+						time.Sleep(time.Second * 2)
```

Comment:
this should be removed. If testing in docker, restart the image.

Reply:
Bummer... but I'm ok with that. This might become a bigger issue once/if we start testing with docker/k8s while moving our services to use uber/fx (none of this was happening pre uber/fx) but that's something to face later